content/docs/atom-guide/dev-guide/render_pipelines.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Render Pipelines"
+description: "You can use pre-built render pipelines or create custom ones in Atom Renderer, the graphics engine integrated into Open 3D Engine (O3DE)."
+toc: false
+---
+
+A *render pipeline* describes a sequence of steps to render a 3D scene to a 2D screen. In **Atom Renderer**, the render pipeline is completely modular and driven by a series of passes. This allows you to customize the render pipeline at a feature level (such as shadows, lighting, and post effects) or by changing the rendering technique completely (such as to Deferred, Checkerboard, or Forward+). By default, Atom implements *Forward+* rendering in its pre-built render pipelines.
```

Comment:
Reply to edit suggestion: "feature level" is a specific term in Direct3D, so I think that term can remain. I wonder, though, if it would be more precise and still accurate to say "...allows you to customize the render pipeline for a specific feature level..."? Otherwise, I'm not sure I understand the nuance of customizing the pipeline _at_ a feature level.

Reply:
Gotcha. Then to avoid confusion, I would definitely avoid the use of the phrase "feature levels" here. I thought you were trying to work that in.